controllers/reviews.js

```diff
@@ -1,5 +1,25 @@
-const { selectReviews } = require('../models/reviews');
+const { lastIndexOf } = require('../db/data/test-data/categories');
+const { selectReviews, selectReviewById } = require('../models/reviews');
 
 exports.sendReviews = (req, res) => {
     selectReviews().then((reviews) => res.status(200).send({ reviews }));
-};
\ No newline at end of file
+};
+
+exports.sendReviewById = (req, res) => {
+    const id = req.params.review_id;
+    if (/^\d+$/.test(id)) {
```

Comment:
I like the idea here to check for an invalid ID, but I would suggest we can reply on SQL to give us an error here instead and catch that. Instead of sending an error in the else block.